PhysicsTools/Heppy/python/analyzers/core/TriggerBitAnalyzer.py

```diff
@@ -9,15 +9,30 @@ class TriggerBitAnalyzer( Analyzer ):
     def __init__(self, cfg_ana, cfg_comp, looperName ):
         super(TriggerBitAnalyzer,self).__init__(cfg_ana,cfg_comp,looperName)
         self.processName = getattr(self.cfg_ana,"processName","HLT")
+        self.processPrescaleName = getattr(self.cfg_ana,"processPrescaleName","RECO")
```

Comment:
please rename to `prescaleProcessName` (and the same for the fallback), which makes more sense